assets/main.css

```diff
@@ -666,6 +666,10 @@ input:not([type=radio]),
   font-size: 16px;
   text-decoration: none;
 }
+.breadcrumbs-container .breadcrumbs-listings .third-listing {
+  font-weight: 700;
```

Comment:
Shouldn't this be 400?

Reply:
it should be bold than the second listing

Reply:
In the cod-theme design true, but according to cod-theme-2 it should be 400